docs/README.md

```diff
@@ -7,7 +7,7 @@ Homebrew installs the stuff you need that Apple didn’t.
 - [Gems, Eggs and Perl Modules](Gems,-Eggs-and-Perl-Modules.md)
 - [Homebrew and Python](Homebrew-and-Python.md)
 
-Please note: Homebrew is not 1.0 yet. Generally it works well, but when
+Please note: Homebrew generally works well, but when
 it doesn’t we’d hope you report the bug.
```

Comment:
Might be just me, but "Please note: Homebrew generally works well, _but_..." reads like "Heads up, this software will blow up in your face." The intent of the original message was to say "heads up, you're using beta software and we'd love to hear the bugs you find." IMHO drop the whole paragraph altogether. Anyway, my $.02.